README.md

```diff
@@ -20,6 +20,11 @@ If you are interested in contributing to the OFN in any capacity, please introdu
 
 Our [GETTING_STARTED](GETTING_STARTED.md) and [CONTRIBUTING](CONTRIBUTING.md) guides are the best place to start for developers looking to set up a development environment and make contributions to the codebase.
 
+### Hacktoberfest :tada:
+
+Are you participating to [Hacktoberfest](https://hacktoberfest.digitalocean.com/)? Go check out our [Welcome New Developers project board][welcome-dev]! We have curated all issues we consider to be a good starting point for new members of the community and categorized them by skills and level of complexity. 
```

Comment:
I'd use "participating in" instead of "participating on" here.

Reply:
Thanks 👍 always struggling with this 🙈